cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,28 @@
+describe('Web app', () => {
+    it('Successfully loads.', () => {
+        cy.visit('/')
+    })
+
+    it("Log in and execute query on private data", () => {
+        cy.visit('/')
+        cy.get('input[name="idp"]').clear().type('http://localhost:8080')
+        cy.contains('Login').click()
+
+        cy.get('input#email').type('hello@example.com');
+        cy.get('input#password').type('abc123');
+        cy.contains('button', 'Log in').click();
+        cy.contains('button', 'Authorize').click();
+
+        cy.contains('Logged in as:')
+
+        cy.contains('A list of my favorite books').click()  
+        cy.contains('"It Ends With Us"')      
+    })
+
+    it("Query on private data unauthenticated", () => {
```

Comment:
Can you also add a test for public data?

Reply:
As of [commit bcb449f](https://github.com/SolidLabResearch/generic-data-viewer/pull/28/commits/bcb449f7d7405b935ab49d3a88eaac3fe4f30f83) this test is available.